src/presentation/test/integration/postTransition/generalPartner/general-partner-check-your-answers-person.test.ts

```diff
@@ -0,0 +1,105 @@
+import request from "supertest";
+import app from "../../app";
+
+import enTranslationText from "../../../../../../locales/en/translations.json";
+import cyTranslationText from "../../../../../../locales/cy/translations.json";
+
+import { appDevDependencies } from "../../../../../config/dev-dependencies";
+import GeneralPartnerBuilder from "../../../builder/GeneralPartnerBuilder";
+import CompanyProfileBuilder from "../../../builder/CompanyProfileBuilder";
+import { GENERAL_PARTNER_CHECK_YOUR_ANSWERS_URL } from "../../../../controller/postTransition/url";
+import { getUrl, setLocalesEnabled } from "../../../utils";
+import { formatDate } from "../../../../../utils/date-format";
+import { GeneralPartner } from "@companieshouse/api-sdk-node/dist/services/limited-partnerships";
+import { CONFIRM_GENERAL_PARTNER_CORRESPONDENCE_ADDRESS_URL, CONFIRM_GENERAL_PARTNER_PRINCIPAL_OFFICE_ADDRESS_URL } from "../../../../controller/addressLookUp/url/registration";
```

Comment:
i think these registration imports are the ones causing the issue with the test

Reply:
Forgot to push the changes made for this.